docs/enterprise/concepts.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Concepts
+sidebarDepth: 2
+description: Learn how the Pomerium Enterprise Console works.
+---
+
+# Concepts
+
+## Namespaces
+
+In the Pomerium Enterprise Console, a **Namespace** is a cornerstone organization unit. They are container objects, that behave similar to a unix directory structure.
+
+In each Namespace, you can define an organizational unit of users and groups with fine-grained access management. This enables teams to self-service the routes and policies pertinent to them. Namespaces can optionally inherit from their parent units.
+
+Namespaces enable:
+
+- Self-Service.
+- Hierarchical policy enforcement (both enforced, and optional),
+- Policy organization.
+- RBAC for the Enterprise Console itself.
+
+Each of these sub-concepts are related and build on each other to form a unified security model.
+
+### Self-Service Capabilities
+
+One of the benefits of an identity-aware access proxy is that, once in place, developers and owners of enterprise applications have an incentive to configure their services to be accessible via the proxy.
+
+Self-service has [several benefits](https://www.usenix.org/system/files/login/articles/login_winter16_05_cittadini.pdf):
+
+- Frees global administrators from continuously modifying the configuration per user requests
+- Encourages service owners to own their configuration fragment
+- Ensures a reasonable compromise between development velocity and security controls
+
+Unlike with a VPN, or network driven access control mechanisms, application owners (with limited access permissions managed through namespaces) can maintain route and policy configuration for their own services, while  higher level operations, security, and identity teams are able to enforce higher level authorization and access policies.
+
+### Hierarchical Policy Enforcement
+
+Hierarchical policy lets administrators enforce high level authorization policy. Policies can be optional (self-select), or mandatory.
+
+Identity and access is organized and controlled with Identity and Access Management (**IAM**) teams via your Identity Provider (**IdP**). These users and groups are read by Pomerium from your IdP, and used to determine top-level Namespace organization.
```

Comment:
```suggestion
Identity and access is organized and controlled with Identity and Access Management (**IAM**) teams via your Identity Provider (**IdP**). These users and groups are read by Pomerium from your IdP, and used to determine top-level Namespace organization.
```

I probably wrote them, but I don't follow those sentences. Maybe just delete?

Reply:
I updated your suggestion with a rewrite of that sentence that I _think_ makes it more clear. Thoughts?

Reply:
👍